chsdi/views/admin.py

```diff
@@ -18,6 +19,11 @@ def admin_kml(context, request):
     table_name = settings.get('geoadmin_file_storage_table_name')
     table_region = settings.get('geoadmin_file_storage_table_region')
     api_url = settings.get('api_url')
+    logging.debug("-----------------------------------------------------------------------")
+    logging.debug(bucket_name)
```

Comment:
Not sure if we leave this. Maybe add the name of the variable whose value is printed

Reply:
In the long run, no real need to keep most of the debug logs in my opinion.